### ISSUE TITLE
[feature request] Add an option to disable scheduler with digdag server

### ISSUE THREAD

**Issue by sonots** [treasure-data/digdag#946]
Currently, digdag server has following options:

* --disable-executor-loop
* --disable-local-agent

But, `--disable-executor-loop` stops both schedule executor and workflow executor.
I want to disable only the schedule executor.

The aim is for development purpose.

I am mainly using `digdag start` with `digdag server` for development because they work slightly different with `digdag run`.
While I am developing dig files, I do not need the scheduler.

**Comment by hiroyuki-sato** [comment]
Hello, @sonots 

If you don't write `schedule:` part in dig file, It doesn't run scheduled tasks.

What do you think to use it until implement `--disable-scheduler`?

**Comment by sonots** [comment]
I want to write `schedule:` part on dig file because it will be written for deployment. I do not want to even comment it out.

My workaround which I come up with is that `digdag push && digdag disable` immediately (although it has a potential risk that the workflow is executed by the scheduler)

**Comment by takashima0411** [comment]
In my workaround, It separates schedule and dig file.

example.
production.dig
```
schedule:
  daily>: 20:00:00
```

local.dig
```
// no schedule settings here
```

sample.dig
```
!include :  env.dig

+some_scheduled_tasks:...
```

In local mode, copy local.dig to env.dig.
In production mode, copy production.dig to env.dig by CI server.

**Comment by sonots** [comment]
fixed via #1048
